dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -858,7 +873,7 @@ internal class DokkaSymbolVisitor(
 
     private fun KtAnalysisSession.isObvious(functionSymbol: KtFunctionSymbol, inheritedFrom: DRI?): Boolean {
         return functionSymbol.origin == KtSymbolOrigin.SOURCE_MEMBER_GENERATED && !hasGeneratedKDocDocumentation(functionSymbol) ||
-                !functionSymbol.isOverride && inheritedFrom?.isObvious() == true
```

Comment:
Due to KT-63741 https://github.com/JetBrains/kotlin/commit/7056ad5325324bb8162db4cef2b5bcf02b3c5483, some fake overrides (e.g. from `Enum`) have become `isOverride=true`. This conjunct was just in case.